src/target/drivers/mcu/stm32/f1/freq.h

```diff
@@ -0,0 +1,10 @@
+#ifndef _DTX_STM32F1_FREQ_H_
+#define _DTX_STM32F1_FREQ_H_
+
+#define AHB_FREQ_MHz  72
+#define APB1_FREQ_MHz 36
+#define APB2_FREQ_MHz 72
+#define ADC_FREQ_MHz   9
+#define FREQ_MHz      AHB_FREQ_MHz
```

Comment:
should we define FREQ_MHZ and derive the other value from this?

Reply:
You can't do that.  the values can be set arbitrarily during clock config.  What we do need to do is to validate that the settings in clock config match our defines.  These values all exist as variables, but they aren't constants in the code, and so can't be optimized.  Since we know the values, defining them as constants simplifies some things

Reply:
okay, but when FREQ_MHZ is fixed to AHB_FREQ? I assume FREQ_MHZ is CPU clock while AHB_FREQ is some bus clock. right?

Reply:
FREQ_MHz is the core CPU frequency.  This is defined as the AHB Frequency at least on the F1.